service-worker/bootstrap.js

```diff
@@ -65,23 +65,6 @@ function precacheLinks(response) {
   })
 }
 
-/**
```

Comment:
We no longer need this.  The CDN should block prefetch requests if needed, but this should no longer be react-storefront's responsibility.